src/Xamarin.Android.Build.Tasks/Microsoft.NET.Workload.Android/WorkloadManifest.in.json

```diff
@@ -24,6 +29,26 @@
       "kind": "sdk",
       "version": "@SDK_PACK_VERSION@"
     },
+    "Microsoft.NET.Runtime.MonoAOTCompiler.Task": {
+      "kind": "sdk",
+      "version": "@MONO_AOT_VERSION@"
+    },
+    "Microsoft.NetCore.App.Runtime.AOT.osx-x64.Cross.android-arm": {
+      "kind": "sdk",
+      "version": "@MONO_AOT_VERSION@"
```

Comment:
I think this versioning is wrong. This manifest file will ship with SDK so the versioning will be done by darc

Reply:
We don't have darc in this repo at all yet. Putting the version number in `Configuration.props` follows our current pattern until that is in place.

@pjcollins will be looking into darc, I think. We can start with getting it to bump the dotnet/sdk version.

Reply:
I think there is bigger problem than that. Let me think about the right setup